cmd/kubeadm/app/cmd/join.go

```diff
@@ -57,6 +57,16 @@ func NewCmdJoin(out io.Writer) *cobra.Command {
 		"(required) Shared secret used to secure bootstrap. Must match the output of 'kubeadm init'",
 	)
 
+	cmd.PersistentFlags().Int32Var(
```

Comment:
Sure, but right now with discovery port is the reason this doesn't exactly make sense.